src/lspc.rs

```diff
@@ -51,6 +54,12 @@ pub enum Event {
         lang_id: String,
         text_document: TextDocumentIdentifier,
     },
+    FormatDoc {
+        lang_id: String,
+        config: LsConfig,
```

Comment:
config should be per lang server. i.e. sent to lspc on `start_lang_server`

Reply:
We should persist it inside struct `LangServerHandler`.

Reply:
A bit weird to persist there but it's certainly better than parsing the settings every time we format the code. Fixed :+1: